test-e2e/src/lib.rs

```diff
@@ -7,12 +7,12 @@ mopro_ffi::app!();
 rust_witness::witness!(multiplier2);
 rust_witness::witness!(multiplier2bls);
 rust_witness::witness!(keccak256256test);
-rust_witness::witness!(hashbenchbls);
+rust_witness::witness!(hashbench_bls);
```

Comment:
I remember it doesn't accept `_` and `-`
though there is no test on `hashbenchbls`
maybe you can just delete this line
```suggestion
```